tools/runme/build.sh

```diff
@@ -111,6 +111,21 @@ _sbt_build() {
   cd "$owd"
 }
 
+_sbt_adb_notebooks() {
```

Comment:
Same here: `adb` is a bad name, should be `e2e`.

Reply:
answered above